Library/Homebrew/formula.rb

```diff
@@ -1690,8 +1690,9 @@ def run_test
     mktemp("#{name}-test") do |staging|
       staging.retain! if ARGV.keep_tmp?
       @testpath = staging.tmpdir
+      @env_home = Pathname.new(Dir.mktmpdir("brew_home-", HOMEBREW_TEMP))
```

Comment:
The `_` and `-` in the same name is making me 😭 :trollface:

Reply:
See, now that you've mentioned it, it's going to bother me too 😬.